lib/generators/shopify_app/install/install_generator.rb

```diff
@@ -53,17 +53,21 @@ def inject_embedded_app_options_to_application
       end
 
       def inject_into_application_controller
+        injection = "  include ShopifyApp::Controller\n"
+        if embedded_app?
+          injection << "  layout \"embedded_app\"\n"
+        end
+
         inject_into_class(
           "app/controllers/application_controller.rb",
           'ApplicationController',
-          "  include ShopifyApp::Controller\n"
+          injection
         )
       end
```

Comment:
I meant about the explicit layout set for esdk.

Including the concerns is nice imo because it gives an easy way to share code and for users to discover the code.